lib/OpenQA/Setup.pm

```diff
@@ -373,43 +346,30 @@ sub load_plugins {
     OpenQA::Setup::update_config($server->config, @{$server->plugins->namespaces}, "OpenQA::WebAPI::Auth");
 }
 
-sub set_secure_flag_on_cookies {
-    my ($controller) = @_;
-    if ($controller->req->is_secure) {
-        $controller->app->sessions->secure(1);
-    }
+sub set_secure_flag_on_cookies ($controller) {
```

Comment:
Since this is a big cleanup patch, convention is to call it `$c` instead of `$controller`.

Reply:
Thanks. I have included this now and moved to https://github.com/os-autoinst/openQA/pull/4419 as a separate PR (as it can be handled separately and also there is no perfect code coverage yet)